nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -10,11 +10,21 @@ let
       options = {
 
         image = mkOption {
-          type = types.str;
+          type = with types; str;
           description = "Docker image to run.";
           example = "library/hello-world";
         };
 
+        imageFile = mkOption {
+          type = with types; nullOr package;
+          default = null;
+          description = ''
+            Path to an image file to load instead of pulling from a registry.
+            If defined, do not pull from registry.
```

Comment:
Maybe it would be nice to say the `image` option still needs to be provided since it is used as the image name.

Reply:
Agreed, fixed in fixup commit.